lib/openfoodfacts.dart

```diff
@@ -538,4 +538,31 @@ class OpenFoodAPIClient {
       return Status(status: 400, error: 'Unrecognized request error');
     }
   }
+
+  /// Returns the Ecoscore description in HTML
+  static Future<String?> getEcoscoreHtmlDescription(
+    final String barcode,
+    final String language,
```

Comment:
Also most other functions provide the optional `QueryType` parameter but I personally think its not that important

Reply:
@MohamedFBoussaid Most other functions also have an optional `User` parameter, that we don't need here: should we put one there too, for consistency between functions? Will that be mandatory in the future?

Regarding `QueryType`, the thing is that the URI doesn't work in test mode. Actually we're not using `'world.openfoodfacts.org'` but `'world-fr.openfoodfacts.org'` (for French). And `'world-fr.openfoodfacts.net' does not seem to work for test environment. @teolemon What is the correct URI for test mode for the Ecoscore Html description?

In the meanwhile I can add an optional `QueryType` parameter and a `TODO` comment.

Reply:
@monsieurtanuki, in that case I agree with you. Let's not add the QueryType. 
Thanks for the changes, all good for me. It can be merged.